lib/sidekiq/middleware/server/freekiqs.rb

```diff
@@ -29,15 +30,15 @@ def call(worker, msg, queue)
                 raise FreekiqException, ex.message
               end
             else
-              Sidekiq.logger.info { "Out of free kiqs for #{msg['class']} job #{msg['jid']}" }
+              Sidekiq.logger.info { "Out of freekiqs for #{msg['class']} job #{msg['jid']}" }
             end
           end
           raise ex
         end
 
         def get_freekiqs_if_enabled(worker, msg, ex)
```

Comment:
Excellent call. I was so focused on getting specs working with that I didn't take a good look at the change made in this class.